pkg/scheduler/nodeinfo/topology_info.go

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
The question is whether we should place topology_info.go and its tests in the "internal" directory or here. The "nodeinfo" directory is not in the internal directory, because other modules use it. We should put topology_info under nodeinfo if we intend to allow other modules (autoscaler, node, etc) to use it and we want to provide backward compatibility. Otherwise, it shouldn't be here.

Reply:
I guess this file was initially written before refactoring. Moved to "internal" package.